cerbero/utils/__init__.py

```diff
@@ -104,6 +105,8 @@ def to_winepath(path):
         path = 'z:\\%s' % path
         return path
 
+def get_wix_wine_prefix():
+    return '/tmp/flu_%s' % str(uuid.uuid4())[:8]
```

Comment:
I would rename flu_ to wix_ this temporary directory has nothing specific to fluendo